### ISSUE TITLE
Download broken in 2.0.0-beta.6

### ISSUE THREAD

**Issue by AhRoon** [simplymemes/crunchyroll-dl#35]
Download is broken. Maybe the Crunchyroll team updated their DRM. Can you fix this?
Thanks.

```
i crunchyroll-dl v2.0.0-beta.6
i Attempting to login...
(node:13744) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'name' of null
    at FormData._getContentDisposition (C:\Users\****\AppData\Roaming\npm\node_modules\crunchyroll-dl\node_modules\form-data\lib\form_data.js:226:40)
    at FormData._multiPartHeader (C:\Users\****\AppData\Roaming\npm\node_modules\crunchyroll-dl\node_modules\form-data\lib\form_data.js:177:33)
    at FormData.append (C:\Users\****\AppData\Roaming\npm\node_modules\crunchyroll-dl\node_modules\form-data\lib\form_data.js:70:21)
    at main (C:\Users\****\AppData\Roaming\npm\node_modules\crunchyroll-dl\index.js:197:15)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
(node:13744) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:13744) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by AhRoon** [comment]
Also, here's my input (In case if needed):
`crunchyroll-dl -i  https://www.crunchyroll.com/saekano-how-to-raise-a-boring-girlfriend- -u **** -p **** --unblocked -l enUS -q 1080p -a -o ":name Episode :ep - :epname (:resolution)"`

**Comment by simplymemes** [comment]
Crunchyroll removed support for the old device I was spoofing, I updated to the latest android version.
Fixed in 2.0.0-beta.7
